inferno-community/fhir-validator-wrapper
# load igs from tgz

This PR refactors `Validator.java` to load IGs as .tgz files, rather than as a folder of profiles/etc.
It also switches the `igs/` directory to contain the .tgz version of US Core 3.1.1, instead of an unzipped version in `/igs/packages`.

To test: 
* run the validator wrapper, and ensure it starts/runs properly (including in Docker-land)
* Run validation against US Core Profiles and make sure they still validate properly
* Add one or more additional IG tarballs from packages.fhir.org, and make sure they all load properly
* Validate against the newly-added IG(s)
